script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -152,6 +152,7 @@ Is he also one of those top-tier vampires――Ancestors?"
 [sha:c378c60f25383d14b2c5d83c812e4a064275acf7]{
 -- Page 160, Offset 22536.
 -- 事実、教会は彼を二十八人目の祖として捉えています。他の祖たちは彼を同位と認めてはいませんが」
+// this one is a title right?
```

Comment:
Hmm not sure, I think just the Twenty-Seven DAA is, but Petrikow can confirm

Reply:
@Petrikow a little help?

Reply:
28th seems best here